api/src/test/resources/provideAndRegRequest-dataset.xml

```diff
@@ -22,7 +22,7 @@
 		voided="false" uuid="bcad3a53-2f78-4db1-9fea-0164156dbbe8" />
 	<patient_identifier_type
 		patient_identifier_type_id="1" name="1.2.3" description="Test description"
-		check_digit="false" validator="" creator="1" date_created="2005-01-01 00:00:00.0"
```

Comment:
It is connected with "org.dbunit.dataset.NoSuchColumnException: PATIENT_IDENTIFIER_TYPE.CHECK_DIGIT". I founded out that it was moved and it isn't used in this place -> https://talk.openmrs.org/t/org-dbunit-dataset-nosuchcolumnexception-patient-identifier-type-check-digit-after-openmrs-2-0-upgrade/7993/4